src/pombot/states.js

```diff
@@ -0,0 +1,11 @@
+/*
+* all the states for a pom
+*/
+
+const states = {
+  RUNNING: 0,
```

Comment:
I like setting states to a string with their name like `RUNNING: 'RUNNING',` that way if you ever have to log the state, `RUNNING` is easier to comprehend than `0`.